packages/amplify-cli-logger/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "amplify-cli-logger",
+  "version": "1.0.0",
+  "description": "Amplify CLI Logger",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "clean": "rimraf lib tsconfig.tsbuildinfo"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aws-amplify/amplify-cli.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/aws-amplify/amplify-cli/issues"
+  },
+  "homepage": "https://github.com/aws-amplify/amplify-cli#readme",
+  "dependencies": {
+    "winston": "^3.3.3",
```

Comment:
I noticed Yathi used https://github.com/aws-amplify/amplify-cli/blob/f6dac5b6d55867e35b28b1c3eec9a6eeb4e4fbe3/packages/amplify-appsync-simulator/src/server/subscription/mqtt-server/client.ts#L7

Any pros/cons to this vs Winston?

Reply:
It's a more mature library with more transports than pino on the other hand pino is smaller but the daily transport file is not that configurable.